modules/swagger-codegen/src/main/resources/swift/Models.mustache

```diff
@@ -10,27 +10,6 @@ protocol JSONEncodable {
     func encodeToJSON() -> AnyObject
 }
 
-public class Response<T> {
-    public let statusCode: Int
-    public let header: [String: String]
-    public let body: T
-
-    public init(statusCode: Int, header: [String: String], body: T) {
-        self.statusCode = statusCode
-        self.header = header
-        self.body = body
-    }
-
-    public convenience init(response: NSHTTPURLResponse, body: T) {
-        let rawHeader = response.allHeaderFields
-        var header = [String:String]()
-        for (key, value) in rawHeader {
-            header[key as! String] = value as? String
-        }
-        self.init(statusCode: response.statusCode, header: header, body: body)
-    }
-}
-
 private var once = dispatch_once_t()
 class Decoders {
```

Comment:
These `Decoders` are part of the `swagger-codegen` project. They're used to generate normal (non-`LoopBackSwift` code). `LoopBackSwiftClientCodgen` does not use them but we can't remove them.